planetterp/config.py.example

```diff
@@ -7,6 +7,14 @@ SECRET_KEY = 'django-insecure-zt7yxn++)bh)j#wzb)ofgd0^scu9rwr85%=3l%6g1zt(cx!t)_
 STATIC_ROOT = None
 ADS_ENABLED = False
 ADMINS = []
+DEBUG = True
+ALLOWED_HOSTS = [
+    "planetterp.com",
+    "api.planetterp.com",
```

Comment:
I wouldn't include any planetterp-specific urls in the example config, leave that for the config on the deploy server